packages/engine/lib/memory/src/shared_memory/segment.rs

```diff
@@ -82,19 +82,17 @@ impl<'id> MemoryId<'id> {
     }
 
     /// Clean up generated shared memory segments associated with a given `MemoryId`.
-    pub fn clean_up<Id: Borrow<Uuid>>(id: Id) -> Result<()> {
+    pub fn clean_up<Id: Borrow<Uuid>>(id: Id) {
         // TODO: macOS does not store the shared memory FDs at `/dev/shm/`. Maybe it's not storing
         //   FDs at all. Find out if they are stored somewhere and remove them instead, otherwise we
         //   have to figure out a way to remove them without relying on the file-system.
-        let shm_files = glob::glob(&format!("/dev/shm/{}_*", Self::prefix(id)))
-            .map_err(|e| Error::Unique(format!("cleanup glob error: {}", e)))?;
+        let shm_files = glob::glob(&format!("/dev/shm/{}_*", Self::prefix(id)));
```

Comment:
This was a result previously I'm assuming because of the `map_err`

Reply:
We are ignoring the error, relevant comments: https://github.com/hashintel/hash/pull/477#discussion_r848332451 https://github.com/hashintel/hash/pull/477#discussion_r848334516
This way if there is a glob error, the whole cleanup doesn't stop.